.github/workflows/build-docker-image.yml

```diff
@@ -48,6 +48,8 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
+        with:
+          submodules: "recursive"
```

Comment:
please remove

Reply:
Pulled in via lifecycle management with updating to devenv-github-workflow package version 1.0.8:
Same handling for Python as well as C++